resources/kubernetes/deployments/abax-minuba.ts

```diff
@@ -43,3 +44,29 @@ export const serverDeployment = new StandardDeployment(
   },
   { providers: [provider] },
 );
+
+export const databaseDeployment = new StandardDeployment(
+  'abax-minuba-db',
+  {
+    image: config.require('db-image'),
+    tag: config.require('tag'),
+    host: config.require('db-host'),
+    logLevel: 'debug',
+    secretEnv: {
+      POSTGRES_USERNAME: pulumi
+        .output(config.require('db-username'))
+        .apply(username => username),
+      POSTGRES_PASSWORD: pulumi
+        .output(config.require('db-password'))
+        .apply(password => password),
+      POSTGRES_DATABASE: database,
+    },
+    ports: [
+      {
+        port: 5432,
+        name: 'db',
+      },
+    ],
+  },
+  { providers: [provider] },
+);
```

Comment:
```suggestion
export const databaseDeployment = new StandardDeployment(
  'abax-minuba-db',
  {
    image: config.require('db-image'),
    tag: config.require('tag'),
    host: config.require('db-host'),
    logLevel: 'debug',
    secretEnv: {
      POSTGRES_USERNAME: pulumi
        .output(config.require('db-username'))
        .apply(username => username),
      POSTGRES_PASSWORD: pulumi
        .output(config.require('db-password'))
        .apply(password => password),
      POSTGRES_DATABASE: database,
    },
    ports: [
      {
        port: 5432,
        name: 'db',
      },
    ],
  },
  { providers: [provider] },
);
```

A standard deployment is a deployment of a docker image, which is not what we want. Furthermore, the username and passwords here should not be configured; they should be retrieved from the database that are created in `abax-minuba-db`.

Did you check `deployment.ts` in unleash/bitwarden?

Reply:
Oh, I see! I was confused about why the `deployment.ts` file pattern was different than `abax-minuba.ts`.

When I looked at it, I wasn't sure if I was gonna need a separate `deployment.ts` file or if I could re-use the pattern written in `abax-minuba.ts`. Since they looked different, I felt more inclined to match the deploy pattern of abax-minuba.

Should I make a deployment (and ingress?). for abax-minuba-db? If so, is it better to make a folder to group the files related to abax-minuba-db together?